### ISSUE TITLE
Allow us to change the font type and size of find-replace widget

### ISSUE THREAD

**Issue by leo-carmo** [microsoft/vscode#101064]
Hi, and please! It's unreadable for my limited eyes:
![image](https://user-images.githubusercontent.com/51187764/85794127-7940c580-b70c-11ea-9ade-6a1f91a41625.png)

I know we are now able to resize the widget horizontally (like pointed out at issues #2657, #20447 and #2220), but that doesn't change the legibility of the inserted characters.

Look how easier it is on Sublime Text to read:
![image](https://user-images.githubusercontent.com/51187764/85794095-6e863080-b70c-11ea-981b-0d1446787eae.png)

There should be a pair of settings like `"editor.FindReplaceWidgetFontFace": "Consolas",` and `"editor.FindReplaceWidgetFontSize": "16",` (the names are just examples).

**Comment by Lucasmateuseng** [comment]
Corcordo com o recurso acima, as letras maiores ajudaria bastante da ux da interface

**Comment by ssbarnea** [comment]
That is clearly a very important accessibility aspect even for English users, as long texts do not fit.

**Comment by ArturoDent** [comment]
@ssbarnea You can drag the left edge of the find widget to expand it.  But it desperately needs a larger font size/letter spacing if you are working with regex's in particular.

**Comment by Plrrb** [comment]
And I think it needs Syntax Highlighting for Regex (Regular Expressions)

**Comment by sweettyler** [comment]
agree that find/replace box should use monospace font only!

**Comment by leo-carmo** [comment]
So, any updates or workarounds? :(

**Comment by agftrading** [comment]
> @ssbarnea You can drag the left edge of the find widget to expand it. But it desperately needs a larger font size/letter spacing if you are working with regex's in particular.

This does not seem to be possible in my VS Code, is there a setting that I need to change or is this perhaps an insiders edition (that am not using yet) feature.  Thanks for any replies.

**Comment by mike-clark-8192** [comment]
Here's a workaround until there's a setting for it. Use the Command Palette (`Ctrl/⌘`+`Shift`+`P`). Choose 'Developer: Open Webview Developer Tools'. In the JavaScript console, run something like this:

```
[...$(".find-widget").getElementsByTagName("textarea")].forEach(x => {const s=x.style; s.fontFamily='monospace'; s.fontSize='14pt';});
```

NB: Never paste or run any JavaScript in the developer console that you don't understand what it does.

**Comment by leo-carmo** [comment]
> Here's a workaround until there's a setting for it. Use the Command Palette (`Ctrl/⌘`+`Shift`+`P`). Choose 'Developer: Open Webview Developer Tools'. In the JavaScript console, run something like this:
> 
> ```
> [...$$(".find-widget textarea")].forEach(x => {const s=x.style; s.fontFamily='monospace'; s.fontSize='15pt';});
> ```
> 
> NB: Never paste or run any JavaScript in the developer console that you don't understand what it does. If you want to make it semi-permanent, you could try: [be5invis.vscode-custom-css](https://marketplace.visualstudio.com/items?itemName=be5invis.vscode-custom-css)

I tried, but then I came across [this](https://stackoverflow.com/questions/67327632/open-webview-developer-tools-not-working-in-vs-code-1-56) which indicates the Webview Developer Tools are...deprecated? I found that after my editor didn't open any new window after trying `Developer: Open Webview Developer Tools` in the command palette. But I could open the window under the command `Developer: Toggle Developer Tools`, and switched to the console tab and inserted your code. Didn't work. 
What am I missing?

**Comment by starball5** [comment]
Related on Stack Overflow: [How do I increase the font size in VS Code find widget?](https://stackoverflow.com/q/78555922/11107541)

**Comment by antonioscript** [comment]
I'm really looking forward to this feature. It's been getting in my way for a while, and accessibility like this would help a lot.

**Comment by mike-clark-8192** [comment]
> I tried, but then I came across [this](https://stackoverflow.com/questions/67327632/open-webview-developer-tools-not-working-in-vs-code-1-56) which indicates the Webview Developer Tools are...deprecated? I found that after my editor didn't open any new window after trying `Developer: Open Webview Developer Tools` in the command palette. But I could open the window under the command `Developer: Toggle Developer Tools`, and switched to the console tab and inserted your code. Didn't work. What am I missing?

You're right, the `Open Webview Developer Tools` item in the command-palette no longer seems to work. However, the `Developer: Toggle Developer Tools` command palette does work for this workaround. The snippet of JavaScript code I provided does work for me, and has for many months across multiple updates. If you are not seeing a change in the font face and font size, we might be talking about a different Find Box. The Find Box that my JS snippet modifies is the "inline" one that overlays your active editor, not the Ctrl+Shift+F (global find across files) box. If you *are* looking at the editor overlay inline find box, maybe do a test by increasing the font size in the JS snippet to make the size change dramatically obvious (the default settings in my JS Snippet do not make the font dramatically larger than the default font, just a little larger.)

**Comment by leo-carmo** [comment]
That worked, [mike-clark-8192](https://github.com/mike-clark-8192)! I used the `Developer: Toggle Developer Tools` instead of the `Open Webview Developer Tools` command. Awesome for readability!

**Comment by leo-carmo** [comment]
@mike-clark-8192 Turns out that js in the console works, but is not permanent. How to make the change persistent?

**Comment by KodywithaK** [comment]
Bumping, this would be a nice accessibility feature

As stated above, `Ctrl/⌘`+`Shift`+`P` > `Developer: Toggle Developer Tools` > `Console` >
```
allow pasting
[...$$(".find-widget textarea")].forEach(x => {const s=x.style; s.fontFamily='monospace'; s.fontSize='15pt';});
```
works almost perfectly **(find widget's width is still tiny, but otherwise great)**
for my use case

**Comment by leo-carmo** [comment]
> Bumping, this would be a nice accessibility feature
> 
> As stated above, `Ctrl/⌘`+`Shift`+`P` > `Developer: Toggle Developer Tools` > `Console` >
> 
> ```
> allow pasting
> [...$$(".find-widget textarea")].forEach(x => {const s=x.style; s.fontFamily='monospace'; s.fontSize='15pt';});
> ```
> 
> works almost perfectly **(find widget's width is still tiny, but otherwise great)** for my use case

Works, but is not persistent across sessions, so... doesn't work actually.

**Comment by alexchexes** [comment]
6 years.
Any chance that this will be addressed?
